model_development/train_image_classifier.py

```diff
@@ -45,11 +45,20 @@ def get_dataloaders(data_dir, batch_size=32):
         train_subset = Subset(dataset, train_ids)
         val_subset = Subset(dataset, val_ids)
         train_loader = DataLoader(
-            train_subset, batch_size=batch_size, shuffle=True, num_workers=4
+            train_subset,
+            batch_size=batch_size,
+            shuffle=True,
+            num_workers=4,
+            transform=data_transforms["train"],
```

Comment:
Can you pass transformations in DataLoader? They seem to not be a defined parameters [here](https://pytorch.org/docs/stable/data.html#torch.utils.data.DataLoader).

In any case, both dataloaders are from subsets of the same dataset, which you define with dataset.ImageFolder(..., data_transforms["train"]). So both have the train transformations.

Reply:
@Sonnyburni